src/cache.c

```diff
@@ -263,7 +263,7 @@ release_subtxn_pinned_caches(SubTransactionId subtxnid, bool abort)
 		if (cp->subtxnid == subtxnid)
 		{
 			/*
-			 * This assert makes sure that that we don't have a cache leak
+			 * This assert makes sure that we don't have a cache leak
```

Comment:
This isn't precisely a spelling error. It's a new feature and I'm still experimenting both with the reporting and the fixes. I'm pretty confident that I haven't messed up any of the fixes...